mainnet/15-wysiwyg-wrong-permissions/evidence/against.md

```diff
@@ -0,0 +1,15 @@
+The proposal Justification says:
+
+> This proposal will initialize the Execute actions permission on the Agent app, assign the Manager role to the Voting app, and assign the permission itself to the Tokens app.
+
+However the blockchain data shows:
+
+> ACL: Create a new permission granting `0x9958466922867D4fc0Da7451c9E717e4ff92D6fE` the ability to perform actions of role �Execute actions� on �Agent� (setting �Voting� as the permission manager)
```

Comment:
```suggestion
> ACL: Create a new permission granting `0x9958466922867D4fc0Da7451c9E717e4ff92D6fE` the ability to perform actions of role "Execute actions" on "Agent" (setting "Voting" as the permission manager)
```

Reply:
Done in https://github.com/aragon/court-precedence-campaign/pull/3/commits/f412eb172d8aee814638bfa7b243d393bb5b7134